core/server/api/canary/redirects.js

```diff
@@ -8,10 +10,20 @@ module.exports = {
         headers: {
             disposition: {
                 type: 'file',
-                value: 'redirects.json'
+                value() {
+                    return redirects.settings.getCurrentRedirectsFilePath()
+                        .then((filePath) => {
+                            return path.extname(filePath) === '.yaml'
+                                ? 'redirects.yaml'
+                                : 'redirects.json';
+                        });
+                }
             }
         },
         permissions: true,
+        response: {
+            format: 'plain'
```

Comment:
What would be a response with this `plan` format look like? Assuming it adjusts to whatever the `redirects` file type is? (don't remember from the top of my head)

Reply:
By default, Ghost server responds in `json` format. 

https://github.com/TryGhost/Ghost/blob/43b7c7b3444aa695a5cc044c65e065ac0262c9ab/core/server/api/shared/http.js#L93-L94

But if we use this format with `yaml` file, the downloaded file is converted to JSON string. It's not what we want. That's why we need to use `plain` text for `yaml` files.